ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,464 +1,476 @@
       neighbor {{ neighbor }} remote-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].remote_as }}
-{%                      elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is defined and router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is not none %}
+{%                 elif router_bgp.vrfs[vrf].neighbors[neighbor].peer_group is arista.avd.defined %}
       neighbor {{ neighbor }} peer group {{ router_bgp.vrfs[vrf].neighbors[neighbor].peer_group }}
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined and router_bgp.vrfs[vrf].neighbors[neighbor].local_as is number %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].local_as is arista.avd.defined and router_bgp.vrfs[vrf].neighbors[neighbor].local_as is number %}
       neighbor {{ neighbor }} local-as {{ router_bgp.vrfs[vrf].neighbors[neighbor].local_as }} no-prepend replace-as
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].description is defined and router_bgp.vrfs[vrf].neighbors[neighbor].description is not none %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].description is arista.avd.defined %}
       neighbor {{ neighbor }} description {{ router_bgp.vrfs[vrf].neighbors[neighbor].description }}
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is arista.avd.defined %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is arista.avd.defined %}
       neighbor {{ neighbor }} ebgp-multihop {{ router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop if router_bgp.vrfs[vrf].neighbors[neighbor].ebgp_multihop is number else none }}
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self is defined and router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self == true %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].next_hop_self is arista.avd.defined(true) %}
       neighbor {{ neighbor }} next-hop-self
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].timers is defined and router_bgp.vrfs[vrf].neighbors[neighbor].timers is not none %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].timers is arista.avd.defined %}
       neighbor {{ neighbor }} timers {{ router_bgp.vrfs[vrf].neighbors[neighbor].timers }}
-{%                      endif %}
-{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].send_community is defined and router_bgp.vrfs[vrf].neighbors[neighbor].send_community == 'None' %}
+{%                 endif %}
+{%                 if router_bgp.vrfs[vrf].neighbors[neighbor].send_community is arista.avd.defined('None') %}
```

Comment:
Refactor line 414 to: `if router_bgp_vrfs[vrf].neighbors[neighbor].send_community is arista.avd.defined and router_bgp_vrfs[vrf].neighbors[neighbor].send_community | lower != "none"` and remove line 412,413.
Alternative do a cli_var if arista.avd.defined and add community if != "none"

Reply:
resolved with a cli builder approach, let me know what you think?